Source/hydro/Castro_mol_hydro.cpp

```diff
@@ -99,9 +100,7 @@ Castro::construct_mol_hydro_source(Real time, Real dt, MultiFab& A_update)
           }
 
 #if (AMREX_SPACEDIM <= 2)
-          if (!Geom().IsCartesian()) {
-            pradial.resize(amrex::surroundingNodes(bx,0),1);
-          }
+          pradial.resize(amrex::surroundingNodes(bx,0),1);
```

Comment:
Was this removal of the !IsCartesian() guard intentional?

Reply:
yes, the code has valgrind errors otherwise.

Reply:
It seems the difference here is that in CTU we only use pradial in the C++ hydro inside !IsCartesian() regions, but we're passing it ca_fourth_single_stage either way.